lib/ecto/query/builder.ex

```diff
@@ -121,11 +121,29 @@ defmodule Ecto.Query.Builder do
     escape_with_type(op_expr, type, params_acc, vars, env)
   end
 
+  def escape({:type, _, [{bitwise_op, _, [_, _]} = op_expr, type]}, _type, params_acc, vars, env)
```

Comment:
I am a bit torn on adding these operations, because in Elixir they work on numbers, and here they work on bitstrings. It feels like a false equivalence to me. So, for now, let's further simplify this PR so it only adds the type but not the new operators? Sorry for the delay.

Reply:
> If you believe it would be better, I can modify this PR to introduce only the Ecto type. Afterwards, we can discuss the possibility of adding bitwise operators.

Yes please!

Reply:
Sure! No problem, I'll do it as soon as possible.